virt_lightning/shell.py

```diff
@@ -227,6 +227,44 @@ def ansible_inventory(configuration, context, **kwargs):
             print(domain.name)  # noqa: T001
 
 
+def ssh_config(configuration, context, **kwargs):
+    conn = libvirt.open(configuration.libvirt_uri)
+    hv = vl.LibvirtHypervisor(conn)
+
+    ssh_host_template = (
+        "Host {name}\n"
+        "     Hostname {ipv4}\n"
+        "     User {username}\n"
+        "     IdentityFile {ssh_key_file}\n"
+    )
+
+    groups = {}
+    for domain in hv.list_domains():
+        for group in domain.groups:
+            if group not in groups:
```

Comment:
That will always not in `groups`, you defined empty dict() on line 241.

Reply:
Fixed, but this snippet came from ansible_inventory function, so same issue there.